wlauto/external/uiauto/src/com/arm/wlauto/uiauto/UxPerfUiAutomation.java

```diff
@@ -16,8 +16,19 @@
 package com.arm.wlauto.uiauto;
 
 import java.util.logging.Logger;
+import android.os.Bundle;
 
 public class UxPerfUiAutomation extends BaseUiAutomation {
+    
+    public Bundle parameters;
+    public String packageName;
+    public String packageID;
```

Comment:
Data members should generally be protected. Is there a reason for this to be public?